unstructured/partition/html.py

```diff
@@ -71,7 +71,11 @@ def partition_html(
     # Verify that only one of the arguments was provided
     exactly_one(filename=filename, file=file, text=text, url=url)
 
-    last_modification_date = None
+    if kwargs.get("last_modification_date"):
```

Comment:
Why did this block and the later block for last modification date get added?

Reply:
With my changes, `partition_epub` doesn't make a call to `convert_and_partition_html` any more and instead directly calls `partition_html` and passes the epub sections to it as text. Because of that, I added calls to `get_last_modified_date_from_file` and `get_last_modified_date` in `partition_epub` and pass those on to `html_partition`. Since the epub info is being pass to `partition_html` as text and `last_modification_date`, is being passed as an argument, I added that logic to get it from the `kwargs`

Reply:
Instead of modifying that here, you'll want logic similar to the following to pull that information from the epub file in `partition_epub`.

https://github.com/Unstructured-IO/unstructured/blob/331c7faf38984d0ce29d920f6ac51d5071f6c0c5/unstructured/partition/html.py#L147-L164